source/deprecations/v1.x.html.md

```diff
@@ -951,6 +958,52 @@ over the native counterparts.
 They will be extracted as an ember addon in case you need to keep using them, but the recomendation now is to
 just use the native array methods or those in libraries like Underscore/Lodash.
 
+#### Ember.ReduceComputedProperty / Ember.ArrayComputedProperty
+
+In addition to `Ember.ReduceComputed` and `Ember.ArrayComputed` you could add a property function call, making use of
+the ReduceComputedProperty and ArrayComputedProperty classes.
+This usage is also deprecated.
+For an example, consider a counter component that displays a total sum of values in an array.
+Using `Ember.ReduceComputedProperty` would show the following:
+
+```js
+totalCount: Ember.reduceComputed({
+  initialValue = 0;
+  addedItem(totalValue, newValue) {
+    return totalValue + newValue;
+  },
+  removedItem(totalValue, newValue) {
+    return totalValue - newValue;
+  }
+}).property('values')
+```
+
+Now that these APIs are deprecated, use the native JavaScript reduce along with the regular computed API:
+
+```js
+totalCount: Ember.computed('values.[]', function() {
+  return this.get('values').reduce((previousValue, currentValue) => {
+    return previousValue + currentValue;
+  })
+})
+```
+
+#### Modifying a Property within `didInsertElement`
+
+Because of changes in the way Ember renders components in 1.13, setting properties within an overridden `didInsertElement` method
+will result in a deprecation warning.
+If you must set a property from within `didInsertElement`, you can schedule the set operation to `afterRender` as follows:
+
+```js
+  didInsertElement() {
+    this._super(...arguments);
+    ...
+    Ember.run.schedule('afterRender', () => {
+      this.set('myValue', value);
+    });
+  }
+```
+
```

Comment:
The correct new approach is to use `didRender`

If that doesn't work for some reason, we should fix it.

In what situations do you need to use `afterRender` in a `didInsertElement` hook in the first place?